core/domain/suggestion_services.py

```diff
@@ -2281,62 +2385,64 @@ def update_translation_review_stats(
             reviewed_translations_count=1,
             reviewed_translation_word_count=content_word_count,
             accepted_translations_count=1,
-            accepted_translations_with_reviewer_edits_count=1 * int(suggestion.edited_by_reviewer),
-            accepted_translation_word_count=content_word_count * int(is_accepted),
+            accepted_translations_with_reviewer_edits_count=1 * int(
```

Comment:
Same concerns in this and other functions in this file, as above.

Reply:
Done